### ISSUE TITLE
Warps,Jails and Rules not enabled on the server

### ISSUE THREAD

**Issue by Jwaffles04** [MarkehMe/FactionsPlus#204]
Whenever I go into the configs to edit them, I save them and restart the server but it just reverts back to what is was before. Chest shop is also effected by this

**Comment by MarkehMe** [comment]
Can you elaborate what yo mean by "Chest shop is also effected by this"? I'm not sure I understand

**Comment by Jwaffles04** [comment]
Well the whole problem is the configurations not saving when I change them and click save

**Comment by MarkehMe** [comment]
This sounds like an issue with your server itself, the files are not saving?

**Comment by Jwaffles04** [comment]
Yeah they aren't saving, everything else is able to save besides factions plus

**Comment by tehrenderr** [comment]
Getting the same issue, I set faction warps to true and it just goes right back to false.

**Comment by DraksterAU** [comment]
Are you using regular spigot 1.9.2?

**Comment by tehrenderr** [comment]
I am not, but I am forced to use this version since it's the only one that supports factions 2.8.7 beta. You guys don't support that on factionsplus beta4 for some reason :/. I also noticed that the newer version of factionsplus overrides essentials /warp and /rules which is really inconvenient.

**Comment by DraksterAU** [comment]
If your running running 1.9 - why not use factions 2.8.6 and same yiueSelf the heartache.

A beta release is a beta release for a reason :)

**Comment by tehrenderr** [comment]
The other versions don't support 1.7

**Comment by DraksterAU** [comment]
Yes they do.... Are you talking about Cauldron/Thermos?

**Comment by DraksterAU** [comment]
Install this if you are - Mark made some fixes we found into a plugin:

https://github.com/MarkehMe/MassiveCoreForgeFixes


Only works up to 2.8.6

**Comment by tehrenderr** [comment]
No other version worked with paperspigot 1.7.10 and factions beta. Every other jar broke except for the 1.9 one.

**Comment by DraksterAU** [comment]
How come your not using regular spigot 1.7.10?

**Comment by DraksterAU** [comment]
Why are you so keen on using the factions beta, which they admit themselves isn't really stable?
